### DISCUSSION TITLE
Tavily

### DISCUSSION THREAD

**Discussion by Shubham-EV** [ItzCrazyKns/Vane#420] [Ideas]
Use tavily API for web search?

**Comment by HoustonMuzamhindo** [comment]
Digging deep into the code to see how we can switch the search engine from SearXng to Tavily. From what I can see, Perplexica consumes JSON results, which is what you will get from Tavily.

**Reply by Shubham-EV** [reply]
but tavily results are more accurate and up to date. How to use it?

**Reply by HoustonMuzamhindo** [reply]
I prefer Tavily for that reason too but it’s not open source like SearXNG. Understandable why Perplexica author went the way he did. I don’t know yet how to use Tavily but theoretically it should be possible.
